Directory.Build.targets

```diff
@@ -25,8 +25,8 @@
 
   <!-- platform version number information -->
   <PropertyGroup Condition="'$(_MauiTargetPlatformIsiOS)' == 'True'">
-    <SupportedOSPlatformVersion>13.0</SupportedOSPlatformVersion>
-    <TargetPlatformMinVersion>13.0</TargetPlatformMinVersion>
+    <SupportedOSPlatformVersion>11.0</SupportedOSPlatformVersion>
+    <TargetPlatformMinVersion>11.0</TargetPlatformMinVersion>
```

Comment:
It looks like the minimum here is going to be 12.2 according to @dalexsoto 

DOTNET_MIN_IOS_SDK_VERSION=12.2
DOTNET_MIN_TVOS_SDK_VERSION=12.2
DOTNET_MIN_MACCATALYST_SDK_VERSION=15.0
DOTNET_MIN_MACOS_SDK_VERSION=12.0

Reply:
Are all of this supposed to _only_ affect Blazor+MAUI apps?

That is what I remember in the meeting, but reading the changes it might affect _all apps_?

Reply:
I think that was when we were going to raise them higher than what the platforms themselves targeted as a min.  Now we are talking about just implicitly setting to the min the platform supports only and trying to better align around what that version is in the future.

MAUI Blazor already requires a higher version than this to work properly, but we talked about not artificially constraining that if the user sets something different (initially we talked about changing what the user set if it was < what we knew would generally work).